src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Runners/MonitorRunnerExtensions.cs

```diff
@@ -84,7 +86,7 @@ public static async Task<HttpClient> CreateHttpClientDefaultAddressAsync(this Mo
 
             string address = await runner.GetDefaultAddressAsync(cancellation.Token);
 
-            return await runner.CreateHttpClientAsync(factory, address, name, cancellation.Token);
+            return await runner.CreateHttpClientAsync(factory, address, name, cancellation.Token, timeout);
```

Comment:
```suggestion
            return await runner.CreateHttpClientAsync(factory, address, name, cancellation.Token, TestTimeouts.HttpApi);
```

I think this should be set to the HttpApi timeout explicitly. While in practice the `timeout` value is probably source from this same static value, the parameter shouldn't be used for both the timeout associated with the creation of the `HttpClient` and the timeout for a HTTP invocation.